src/Evolution/Systems/Burgers/Fluxes.cpp

```diff
@@ -13,9 +13,11 @@
 
 // IWYU pragma: no_forward_declare Tensor
```

Comment:
Please remove the "explanation" in the commit message because to me it just is confusing. Alternative, I would say make the commit message just "Doxygen cond Burgers Fluxes.cpp" since that's not a "hmm what's going on here" because we are fairly familiar with Doxygen doing weird things with the cpp files.